test/Interop/Cxx/value-witness-table/Inputs/basic.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TEST_INTEROP_CXX_DESTRUCTOR_INPUTS_BASIC_H
+#define TEST_INTEROP_CXX_DESTRUCTOR_INPUTS_BASIC_H
+
+struct HasUserProvidedDestructor {
+  int *value;
+  ~HasUserProvidedDestructor() { *value = 42; }
+};
+
+struct EmptyDestructor {
```

Comment:
Could you rename this type to something more specific, like you named other types?

Reply:
Renamed to `HasEmptyDestructorAndMemberWithUserDefinedConstructor`.